src/pages/Login/Login.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import LoginInput from './components/LoginInput';
+import styled from 'styled-components';
+
+function Login() {
+  return (
+    <Container>
+      <article>
+        <Title>
+          <h2>
+            <img alt="cream로고" src="/images/login_logo.png" />
+          </h2>
+        </Title>
```

Comment:
```suggestion
    <Container>
      <img alt="cream로고" src="/images/login_logo.png" />
```
1. 불필요한 depth가 있습니다! html 태그의 depth는 최대한 줄여주시는게 좋습니다!
2. Container에 flex-direction: column을 주면 article 태그를 없애도 됩니다!
3. 마찬가지로 Title 컴포넌트와 h2태그 없이 바로 img태그를 넣어줘도 되겠죠?
4. img 스타일트컴포넌트를 만들어서 수정해주세요 :)

Reply:
넵 수정했습니다! 도현님 추가로 질문이 있습니다만.. 뭔가 컴포넌트도 semantic하게 사용하려고 수정하기 전처럼 해봤었는데, 굳이 저렇게 할 필요는 없는건가요?? 궁금해서 여쭙습니다!

Reply:
semantic 태그는 스타일드 컴포넌트에도 사용할 수 있습니다!
const Container = styled.article`` 이런식으로요
수정의 목적은 불필요한 태그들을 줄이기 위함입니다!